client/python/openlineage/client/client.py

```diff
@@ -245,3 +258,60 @@ def _http_transport_from_url(
                 session=session,
             ),
         )
+
+    def _alias_env_vars(self) -> None:
+        default_transport_name = self.DEFAULT_URL_TRANSPORT_NAME.upper()
+        if url := os.environ.get("OPENLINEAGE_URL"):
+            os.environ[f"OPENLINEAGE__TRANSPORT__TRANSPORTS__{default_transport_name}__TYPE"] = "http"
```

Comment:
Maybe we should check if this variable name is already taken and then raise some kind of error? Just to be explicit and not overwrite something by accident, debugging this could be hard

Reply:
Not sure if raising exception is the right approach. I did warning log and just skip aliasing if the transport with default name already exists. Added test as well.